.github/workflows/automated-deploy-dev.yml

```diff
@@ -1,5 +1,5 @@
 # .github/workflows/terraform-dev
-name: 'Development CI Feature to Main'
+name: 'Z-AUTOMATED Deploy: Dev'
```

Comment:
Should there be something in between Z-Automated and Deploy

"Z-AUTOMATED: Deploy - Dev"  ?